did_doc_sov/src/extra_fields/didcommv1.rs

```diff
@@ -56,12 +68,25 @@ impl ExtraFieldsDidCommV1Builder {
         self
     }
 
+    pub fn set_accept<I>(mut self, accept: I) -> Self
+    where
+        I: IntoIterator<Item = AcceptType>,
+    {
+        self.accept = accept.into_iter().collect();
+        self
+    }
```

Comment:
When working on the PR I made `accept` a `HashMap` and this is a leftover. I will remove it. Not sure about the overhead, I would expect it would not be hard for the compiler to remove these redundant calls.